README.md

```diff
@@ -1 +1,70 @@
-# Conference
\ No newline at end of file
+![](./images/screenshot.jpg)
+
+# Conference Website
+
+> Meetups and professional conferences have grown into every possible industry around the world. The Internet makes it much easier for people to connect and attend these conferences. 
+> 
+>The goal of a conference website is two-fold: 
+>1. Inform & spread awareness
+>2. Capture ticket sales
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo
+
+[Live Demo Link](https://toliboff.github.io/Conference/)
+
+## Getting Started
+
+**This is ready for use Conference website**
+**Modify this file to match your project, remove sections that don't apply.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+  - NodeJS
+### Setup
+- Make sure you have `Node NPM` installed on your computer
+- Clone or download this repo on your machine
+- Enter project directory
```

Comment:
- [ ] Also mention how to clone like `git clone https://github.com/yourrepo.git` and how to enter in your repository like `cd yourrepo`. Think about the person who don't have technical knowledge.